packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1184,8 +1189,31 @@
                 {
                     "type": "proxiedSessionId",
                     "required": false
+                },
+                {
+                    "type": "timestamp",
+                    "required": false
                 }
             ]
+        },
+        {
+            "name": "ide_heartbeat",
```

Comment:
`toolkit_heartbeat` may fit better. ide_ is for non-extension evens (from the ide)